ledger-tool/src/bigtable.rs

```diff
@@ -92,12 +94,71 @@ async fn blocks(starting_slot: Slot, limit: usize) -> Result<(), Box<dyn std::er
     Ok(())
 }
 
+async fn compare_blocks(
+    starting_slot: Slot,
+    limit: usize,
+    cred_path: Option<String>,
+) -> Result<(), Box<dyn std::error::Error>> {
+    if let Some(credential) = &cred_path {
+        assert!(!credential.is_empty());
+    }
+
+    let bigtable_def = solana_storage_bigtable::LedgerStorage::new(false, None, None)
```

Comment:
What's a `bigtable_def`?

Reply:
def => default. we have setup a default bigtable from  $GOOGLE_APPLICATION_CREDENTIALS. I will think about other name